assets/styles/importer.less

```diff
@@ -184,6 +184,14 @@ td h3 {
   color: green;
 }
 
+[ng\:cloak], [ng-cloak], .ng-cloak {
```

Comment:
[It's apparently possible but a bit messy](https://stackoverflow.com/questions/14968690/sending-event-when-angular-js-finished-loading). As an alternative, we could exclude the pokeball from the ng-cloak; that way the pokeball will start spinning before the angular stuff is sorted.